### ISSUE TITLE
Uncaught TypeError: Cannot read property 'path' of undefined

### ISSUE THREAD

**Issue by lcd1232** [pghilardi/atom-python-virtualenv#17]
[Enter steps to reproduce:]

1. Open console
2. Choose 'Virtualenv: Deactivate'

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [atom-python-virtualenv](https://github.com/pghilardi/atom-python-virtualenv) package 0.10.0

### Stack Trace

Uncaught TypeError: Cannot read property 'path' of undefined

```
At /Users/lcd1232/.atom/packages/atom-python-virtualenv/lib/virtualenv-manager.coffee:86

TypeError: Cannot read property 'path' of undefined
    at VirtualenvManager.module.exports.VirtualenvManager.getPathForEnv (/packages/atom-python-virtualenv/lib/virtualenv-manager.coffee:86:17)
    at VirtualenvManager.module.exports.VirtualenvManager.deactivate (/packages/atom-python-virtualenv/lib/virtualenv-manager.coffee:100:52)
    at /packages/atom-python-virtualenv/lib/init.coffee:55:15)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at Object.didConfirmSelection (/app.asar/node_modules/command-palette/lib/command-palette-view.js:106:29)
    at SelectListView.confirmSelection (/app.asar/node_modules/atom-select-list/src/select-list-view.js:292:20)
    at /app.asar/node_modules/atom-select-list/src/select-list-view.js:64:14)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -0:38.8.0 core:backspace (input.hidden-input)
     -0:37.7.0 core:move-down (input.hidden-input)
     -0:37.1.0 core:confirm (input.hidden-input)
     -0:37.1.0 virtualenv:select (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
  8x -0:35.5.0 core:backspace (input.hidden-input)
     -0:30.8.0 core:confirm (input.hidden-input)
     -0:10.4.0 command-palette:toggle (input.hidden-input)
     -0:07.4.0 core:move-down (input.hidden-input)
     -0:07.0 core:confirm (input.hidden-input)
     -0:07.0 virtualenv:select (input.hidden-input)
     -0:05.9.0 editor:consolidate-selections (input.hidden-input)
     -0:05.9.0 core:cancel (input.hidden-input)
     -0:05.4.0 intentions:highlight (input.hidden-input)
     -0:04.8.0 command-palette:toggle (input.hidden-input)
     -0:02.2.0 core:confirm (input.hidden-input)
     -0:02.2.0 virtualenv:deactivate (input.hidden-input)
```

### Non-Core Packages

```
atom-python-virtualenv 0.10.0 
atom-typescript 11.0.3 
autocomplete-python 1.8.63 
busy-signal 1.4.1 
intentions 1.1.2 
kite 0.23.0 
language-nginx 0.7.3 
linter 2.1.4 
linter-flake8 2.2.1 
linter-ui-default 1.2.4 
```

**Comment by pghilardi** [comment]
@lcd1232 ,

I think that this bug was solved in the latest version: 0.11.0. 

Could you check?

Thanks

**Comment by lcd1232** [comment]
@pghilardi ,
Yes, you are right.
